opensuse-tomcat-image/src/main/docker/Dockerfile

```diff
@@ -50,7 +45,7 @@ COPY setenv.* $TOMCAT_BIN_DIR/
 
 # Copy a new lib folder with thirdparty jars for logback implementation to tomcat lib folder
 COPY /maven/lib/*.jar $TOMCAT_LIB_DIR/
-RUN mv $TOMCAT_LIB_DIR/tomcat9-slf4j-logback-9.0.26.jar  $TOMCAT_BIN_DIR/tomcat-juli.jar
+RUN mv $TOMCAT_LIB_DIR/caf-logging-tomcat-juli-3.7.0-scmod-7821-new-SNAPSHOT.jar  $TOMCAT_BIN_DIR/tomcat-juli.jar
```

Comment:
You can't include your own version number like this.  How could the build process possibly know to update it for branch builds or for release builds?

Reply:
version number removed.